### PR TITLE
add background-color and window options

### PR DESCRIPTION
**Pull request by xn**

### CODE REVIEW

**Comment by alexanderturinske** [comment]
Yes plz! 👍 @imlucas

**Comment by xn** [comment]
done

**Comment by imlucas** [comment]
@MarshallOfSound @malept Would this logically conflict with #25?

File: README.md [hunk 1]
```diff
@@ -99,6 +104,8 @@ Disk image format. [Default: `UDZO`].
 - `ULFO` :arrow_right: lzfse-compressed image (macOS 10.11+ only)
 
 
+
+
```
Comment (malept) [inline README.md]: Seems unnecessary, there's already whitespace there.

File: usage.txt [hunk 2]
```diff
@@ -10,8 +10,10 @@ Options:
   --icon=<path>        Path to the icon file that will be the app icon in the DMG window.
   --icon-size=<px>     How big to make the icon for the app in the DMG. [Default: `80`].
   --background=<path>  Path to a PNG image to use as the background of the DMG.
+  --background-color=<color>  Background color (accepts css colors).
   --format=<format>    Disk image format. [Default: `UDZO`].
   --debug              Enable debug messages.
   --overwrite          Overwrite any existing DMG.
+  --window="<object>"  Window size and position options. ex: "{size: {width: 100, height: 100}, position" {x: 0, y: 0}}"
```
Comment (malept) [inline usage.txt]: That's not really a great way of handling nested params in a CLI interface. I believe minimist has support for options like `--window.size.width=100`.

Comment (xn) [reply usage.txt]: Actually, that documentation is plain wrong there. I overlooked you were using minimist.

**Comment by xn** [comment]
As far as I understand it, you don't specify integers. From minimist's README:
```Numeric-looking arguments will be returned as numbers unless opts.string or opts.boolean is set for that argument name.```

Checking the docs, the tests and the codebase I don't see a sign of it.

If it does exist, please show me where so I can reference and update.

**Comment by malept** [comment]
Sorry, I was thinking of `yargs-parser`, which has a `number` option.

**Comment by malept** [comment]
That being said, it may not hurt to do a type check on the values.

**Comment by xn** [comment]
With a quick glance and as it stands there seems to be no type checking on the other args. Feels like a separate feature and get them all.

**Comment by MarshallOfSound** [comment]
This will be handled by the catch all property in #32

**Pull request closed without merging**